common/src/main/java/com/wynntils/wc/utils/WynnItemMatchers.java

```diff
@@ -25,6 +25,8 @@ public class WynnItemMatchers {
     private static final Pattern ITEM_RARITY_PATTERN =
             Pattern.compile("(Normal|Set|Unique|Rare|Legendary|Fabled|Mythic)( Raid)? (Item|Reward).*");
     private static final Pattern DURABILITY_PATTERN = Pattern.compile("\\[(\\d+)/(\\d+) Durability\\]");
+    public static Pattern POWDER_PATTERN =
+            Pattern.compile("§[2ebcf8].? ?(Earth|Thunder|Water|Fire|Air|Blank) Powder ([IV]{1,3})");
```

Comment:
(As a side note, I spent longer than I care to admit trying to think about if it was possible to express roman numerals properly in RE. :-))